README.md

```diff
@@ -22,3 +22,15 @@ based on [adhocracy 4](https://github.com/liqd/adhocracy4).
     make install
     make fixtures
     make watch
+
+### Use postgresql database for testing:
+run the following command once:
+```
+make create-postgres
+```
+to start the testserver with postgresql, run:
+```
+export DATABASE=postgresql
+make start-postgres
+make watch
```

Comment:
I had to also do 
`python manage.py migrate`
Without it, I get `ProgrammingError: relation "django_session" does not exist`. 
Maybe add this here?

Reply:
that should be solved if you do `make install`, that shouldn't have to go here

Reply:
Oh, sorry, my mistake - I did it in the order in the readme.